docs/release-notes/v13.1.0.md

```diff
@@ -0,0 +1,165 @@
+---
+id: v13.1.0
+title: Version 13.1.0 Release Notes
+sidebar_label: v13.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_August 29th, 2024_
+
+:::info
+Mia-Platform Console v13.1.0 is **now in Preview** and will be generally available on September 12th.
+
+Console SaaS users can try out v13.1.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### Breaking Changes
+
+The body of the `PUT /api/extensibility/tenants/{tenantId}/extensions` used to register extensions has been modified and simplified, along with the manifest to be sent using the miactl `extensions apply` command.
+
+:::danger

```

Comment:
this is not a danger, is a interesting note about the fact that it is not breaking yet, i'd change this to a more positive banner (e.g. a tip?)